friskby/GW/air_in_bergen.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+
+import requests
+import timestring
+#  import re
+#  import time
+from datetime import date
+import psycopg2
+from friskby.GW.friskby_gw import FriskByGW
```

Comment:
None of the imports `requests, timestring, date` or `psycopg2` are actually used - remove them.